### PR TITLE
Переверстать карточку заявки: действие в центре, единая лента

### PR DESCRIPTION
**Pull request by Antropophag**
## Summary

Реализует вариант E из ранее обсуждённого с пользователем макета (артефакт «Комментарии vs панель действий — сравнение макетов»):

- **Действие в центре.** Главное действие пользователя по заявке (назначение исполнителя, начало работы, загрузка отчёта, экспертное заключение, решение СБ, переназначение эксперта и т. д.) теперь крупный блок в начале основной колонки вместо мелких элементов в узкой правой панели. У зрителя без доступного действия блок не показывается — сразу видна лента. Несколько одновременно доступных действий (например, руководителю ИЦ) показываются друг под другом, разделённые линией.
- **Единая лента.** История переходов статуса и обсуждение объединены в одну хронологическую ленту (`buildFeed` в `registry.js`), как в таймлайне GitHub PR — вместо разрозненных вкладки «Обсуждение» и отдельной модалки «История». Видимость документов (DOC-003/ACL-002) лента не затрагивает — показывает только факт события, не раскрывает содержимое.
- **Правая колонка — чистая справка.** Статус (исполнитель/эксперт/отметка СБ/цветовая метка WF-009), объект испытаний (бывшая «Общая информация»), документы — без единой формы действия.

### Побочная находка (задокументирована, не в объёме этой задачи)

`gmdate('Y-m-d H:i:s.u')` в PHP **всегда** возвращает `.000000` — `gmdate()`/`date()` работают с целочисленным Unix-timestamp и не поддерживают микросекунды, несмотря на то что формат `.u` синтаксически принимается. Обнаружено при тестировании ленты: несколько действий подряд в одну секунду не всегда сортировались в реальном порядке между разными `kind` (`transition`/`assignment`), так как SQL `ORDER BY occurredAt DESC, kind DESC, id DESC` при равных `occurredAt` группирует по `kind`. `buildFeed` реверсирует каждый источник перед merge-сортировкой — это даёт верный порядок внутри одного `kind` и разумный практический компромисс для остальных случаев. Полный фикс заведён отдельно — issue #86.

## Test plan

- [x] Новые/обновлённые unit-тесты `registry.test.js` (`buildFeed`, включая edge case с одинаковым timestamp) — зелёные
- [x] `npm run lint` / `npm test` (63 теста) / `npm run build` — зелёные
- [x] `docker compose up -d --build` + `sh scripts/smoke.sh` — зелёные
- [x] Playwright `npx playwright test` (2 теста) — зелёные
- [x] Визуальная проверка скриншотами: руководитель ИЦ, эксперт, сотрудник СБ без доступного действия
- [x] `docs/user/request-card.md` обновлён под новую структуру карточки

🤖 Generated with [Claude Code](https://claude.com/claude-code)

https://claude.ai/code/session_01MdoygMzQ45xsa4W6jBA4rX

### CODE REVIEW

File: frontend/src/registry.js [hunk 1]
```diff
@@ -96,23 +96,46 @@ const HISTORY_LABELS = {
...
+// удобно для постраничной подгрузки старых записей), а лента должна читаться
+// по возрастанию времени, как переписка. Array.prototype.sort стабилен, но
+// секундная точность occurredAt/createdAt означает, что несколько событий
+// подряд (например, быстрая смена статусов) могут получить одинаковый
+// sortAt — без предварительного разворота каждого источника стабильная
+// сортировка сохранила бы их неверный DESC-порядок внутри одной секунды.
+export function buildFeed(history, comments) {
+  const orderedHistory = [...history].reverse()
+  const orderedComments = [...comments].reverse()
+  return [...orderedHistory, ...orderedComments].sort((a, b) => new Date(a.sortAt) - new Date(b.sortAt))
```
[the remark this answers is not in the document]
Comment (Antropophag) [reply frontend/src/registry.js]: Хорошая находка, исправлено в d0241b5: comments больше не реверсятся повторно в buildFeed() — backend (queryCommentsPage) уже возвращает их в ASC через array_reverse() после ORDER BY id DESC. Реверс остаётся только для history — единственного источника, отдаваемого DESC. Добавлен unit-тест на этот конкретный tie-break случай.

File: frontend/src/App.vue [hunk 2]
```diff
@@ -1145,33 +1150,108 @@ onMounted(bootstrapAuth)
...
                 <button v-if="selected.commentsPage?.hasMore" class="secondary" :disabled="olderCommentsLoading" @click="loadOlderComments">{{ olderCommentsLoading ? 'Загрузка…' : 'Показать предыдущие' }}</button>
-                <div v-for="comment in selected.comments || []" :key="comment.id" class="comment"><span class="avatar small">●</span><div><b>{{ comment.author }}</b><time>{{ comment.createdAt }}</time><p>{{ comment.body }}</p></div></div>
-                <p v-if="!selected.comments?.length" class="placeholder-copy">Комментариев пока нет.</p>
+                <div class="stream">
+                  <div v-for="entry in feed" :key="`${entry.type}-${entry.id}`" class="entry" :class="{ system: entry.type === 'milestone' }">
+                    <span class="avatar small" :class="{ 'blue-avatar': entry.type === 'comment' }">●</span>
+                    <div class="entry-body">
+                      <template v-if="entry.type === 'milestone'">
+                        <div class="entry-head"><b>{{ entry.actor }} — {{ entry.description }}</b><time>{{ entry.occurredAt }}</time></div>
+                      </template>
```
[the remark this answers is not in the document]
Comment (Antropophag) [reply frontend/src/App.vue:1236]: Исправлено в d0241b5: milestone-события в ленте снова показывают ruleId рядом со временем.
